PandaSDK.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |spec|
   #
 
   spec.name         = "PandaSDK"
-  spec.version      = "0.0.8"
+  spec.version      = "0.1.0"
```

Comment:
fix version - it is wrong

Reply:
why? yours was `0.0.9`, then should be `0.1.0`.
or `0.0.10`?

Reply:
will be `0.0.9`